pulsar-client-api/src/main/java/org/apache/pulsar/client/api/Schema.java

```diff
@@ -213,6 +213,16 @@ default void configureSchemaInfo(String topic, String componentName,
         return DefaultImplementation.newProtobufSchema(SchemaDefinition.builder().withPojo(clazz).build());
     }
 
+    /**
+     * Disable SchemaInfo.
+     *
+     * @param schema instance
+     * @return a Schema instance of bytes type
+     */
+    static <T> Schema<T> disabled(Schema<T> schema) {
```

Comment:
I think this option to disable the schema info should be made, if needed, in a separate PR.

Reply:
Thank you, you are right, I will restart a pr to fix the problem of healthcheck.